endpoints/kubernetes/grpc-bookstore.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  ports:
+  # Port that accepts JSON/HTTP requests over HTTP.
+  - port: 80
+    protocol: TCP
+    name: http
+  # Port that accepts gRPC and JSON/HTTP2 requests over HTTP.
+  - port: 9000
+    protocol: TCP
+    name: http2
+  selector:
+    app: esp-grpc-bookstore
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-grpc-bookstore
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```

Comment:
I think we should follow best practices where we can, even if it means more maintenance. http://kubernetes.io/docs/user-guide/config-best-practices/ People often take samples as-is without reading docs or notices. We shouldn't provide samples for testing only, when it is simple enough to have the actual version without any caveats.

That said, for docker images it is typical to have hierarchical tags, sort of like semantic versioning. ESP should have a major version tag, such as `:1` that points to the latest point release, e.g. `:1.2`, and is updated with new point releases. It would be fine to have a higher-level major version tag here.